src/main/java/build/buildfarm/worker/Worker.java

```diff
@@ -473,6 +473,7 @@ public static void main(String[] args) throws Exception {
     Path configPath = Paths.get(args[0]);
     InputStream configInputStream = Files.newInputStream(configPath);
```

Comment:
The try-with-resources makes this (and most scenarios I want to use it in) awkward when I really wanted to just release the resource early, but now I've got a worker that needs to be declared outside the block, I've got to check for null, etc etc.  I'll put the block in, with the manual close, proceeding with the other operations.